### PR TITLE
Edge-Node-Cluster VMI additions

### PR DESCRIPTION
**Pull request by andrewd-zededa**
Track VirtualMachineInstance for eve VMs.

### CODE REVIEW

**Comment by andrewd-zededa** [comment]
@naiming-zededa @xyuria-zededa Please review as time allows.

**Review by naiming-zededa — APPROVED** [review]
LGTM

File: proto/info/edge_node_cluster.proto [hunk 1]
```diff
@@ -269,3 +272,37 @@ enum KubeComp {
...
+  // Name of the VirtualMachineInstance
+  string name = 1;
+
+  // Status (VirtualMachineInstance.Status.Phase)
+  KubeVMIStatus status = 2;
+
+  // Creation Time of the VirtualMachineInstance
+  google.protobuf.Timestamp creation_time = 3;
+
+  // LastTransitionTime is the last time the Ready Condition was true
```
Comment (eriknordmark) [inline proto/info/edge_node_cluster.proto:300]: Does "the Ready Condition" refer to the is_ready field below? If so it makes sense to update this comment.
And I'd say "was set to true" and not "was true".  Would the timestamp also be updated when it is set to false?

**Review by eriknordmark — APPROVED** [review]
LGTM apart from a comment question/comment.

File: proto/info/edge_node_cluster.proto (hunk 1 above)
Comment (andrewd-zededa) [reply proto/info/edge_node_cluster.proto]: I just fixed the comments the current comment does leave a hole.  The timestamp should be updated for both ->ready and ->not-ready transitions.

**Comment by andrewd-zededa** [comment]
> LGTM apart from a comment question/comment.

Addressed it in the latest push.

**Pull request merged**